src/main/cpp/Robot.cpp

```diff
@@ -72,7 +75,8 @@ void Robot::TeleopInit() {
     m_autonomousCommand = nullptr;
   }
 
-  m_container.StartIdling();
+  // m_container.StartIdling();
```

Comment:
Can we start idling again now that the lambda is removed?

Reply:
There might be an issue with calling LEDs at the very start of init. Will hold off on adding back for now